doctr/models/utils/pytorch.py

```diff
@@ -116,7 +123,10 @@ def export_classification_model_to_onnx(model: nn.Module, model_name: str, dummy
         input_names=['input'],
         output_names=['logits'],
         dynamic_axes={'input': {0: 'batch_size'}, 'logits': {0: 'batch_size'}},
-        export_params=True, opset_version=13, verbose=False
+        export_params=True,
+        opset_version=14,
```

Comment:
Perhaps a comment on the opset_version? Is there a reason to upgrade? (ideally a comment saying why we need at least this version is perfect for future contributors)

Reply:
Mh there is not directly a reason .. we only should keep the latest opset version for future models (higher opset version = more operators supported) :)

Reply:
Alright but then according to https://pytorch.org/docs/master/onnx.html#torch.onnx.export we should match the max opset version of the oldest PyTorch version that we support no?

Reply:
Mh.. yes i agree :) 👍